example_settings.ini

```diff
@@ -118,6 +118,31 @@ email_path =
 auth_file =
 worksheet_key =
 
+[header_column]
```

Comment:
sorry I copied the example_settings.ini file to every branch since I thought it will be convenient